src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -495,6 +495,26 @@ public object PrivateData
         [ValidateNotNullOrEmpty]
         public string ReleaseNotes { get; set; }
 
+        /// <summary>
+        /// Specify that the module is prerelease.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ValidateNotNullOrEmpty]
+        public string Prerelease { get; set; }
+
+        /// <summary>
+        /// Specify whether the module requires explicit user acceptance for install/update/save.
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public SwitchParameter RequireLicenseAcceptance { get; set; }
```

Comment:
I'd expect that default is true. 
@SteveL-MSFT what do you think?

Reply:
In anticipation I set the default to true. :)